### PR TITLE
Update CWS best practices page

### PR DESCRIPTION
**Pull request by awfuchs**
Fixes b/x 186578963

Removed old web-app descriptions and categories; added up-to-date copy describing attributes of high-quality extensions.

[Best practices](https://deploy-preview-794--developer-chrome-com.netlify.app/docs/webstore/best_practices/) preview site

### CODE REVIEW

File: site/en/docs/webstore/best_practices/index.md [hunk 1]
```diff
@@ -9,10 +9,33 @@ description: >
...
-## Design a great app
+## Design a high-quality extension
+
+We encourage you to develop extensions that are of high quality. High-quality extensions conform to
+standards of performance, security, and user experience, as summarized by the following guidelines:
+
+* **Compliance:** Does the extension comply with our [program policies]? Extensions that violate any
+of these policies will not be considered for a badge and may be removed from the store.
+
+* **Manifest V3:** Is the extension built on Manifest V3? Manifest V3 reflects the aims of our
```
Comment (lsauser) [inline site/en/docs/webstore/best_practices/index.md:20]: can we change MV3 to not being a requirement, but highly recommended?

File: site/en/docs/webstore/best_practices/index.md [hunk 2]
```diff
@@ -9,10 +9,33 @@ description: >
...
 matures and we learn from developers' experiences, these recommendations will be updated.
 
-## Design a great app
+## Design a high-quality extension
+
+We encourage you to develop extensions that are of high quality. High-quality extensions conform to
+standards of performance, security, and user experience, as summarized by the following guidelines:
+
+* **Compliance:** Does the extension comply with our [program policies]? Extensions that violate any
+of these policies will not be considered for a badge and may be removed from the store.
```
Comment (lsauser) [inline site/en/docs/webstore/best_practices/index.md:18]: need to remove this mention

File: site/en/docs/webstore/best_practices/index.md (hunk 1 above)
Comment (awfuchs) [reply site/en/docs/webstore/best_practices/index.md:20]: Made it should, not must.

File: site/en/docs/webstore/best_practices/index.md (hunk 2 above)
Comment (awfuchs) [reply site/en/docs/webstore/best_practices/index.md]: It is removed.

File: site/en/docs/webstore/best_practices/index.md [hunk 3]
```diff
@@ -9,10 +9,33 @@ description: >
...
+
+* **Security:** Is the extension safe for users? Make sure your extension does not pose security
+  threats, does not use deceptive installation tactics and meets Chrome Web Store's data privacy
+  requirements (see this [FAQ][user-data-faq]).
+
+* **Performance:** Does the extension function at an outstanding level? High-quality extensions
+  don't just perform their intended action, but perform at an extremely high level. 
+
+* **User experience:** Is the extension a joy to use? Make sure your extensions are easy to
+  install and manage, with a good-looking, intuitive and seamless user experience.
```
Comment (dotproto) [inline site/en/docs/webstore/best_practices/index.md:32]:
```suggestion
* **User experience:** Is the extension a joy to use? The extension itself should provide a
  good-looking, intuitive, and seamless user experience while also respecting the end user's
  privacy.
```

I wanted to acknowledge that some extensions currently use content scripts to inject extension UI on all pages. While that may provide a user good user experience for some users, others may not be comfortable giving the extension that much data access.

File: site/en/docs/webstore/best_practices/index.md [hunk 4]
```diff
@@ -9,10 +9,33 @@ description: >
...
+
+* **Store listing:** Does the store listing page provide quality and well-designed images (icon,
+  tile, marquee, and screenshots)? Images should not be blurry or too busy. Additionally, the store
+  listing page should be complete with all fields filled out (see [Supplying images] [supplying-images]
+  and [Completing your store listing] [completing-listing] for more listing tips).
```
Comment (dotproto) [inline site/en/docs/webstore/best_practices/index.md:37]:
```suggestion
* **Store listing:** The extension's [Chrome Web Store listing][completing-listing] should sets the
  user's expectations and clearly communicates what the extension does.   All image assets (icon,
  tile, marquee, and screenshots) [should be provided][supplying-images]. Images should not be
  blurry or too busy. [Privacy information][dashboard-privacy] (permissions justifications, the extension's privacy policy, data use disclosures, etc.) must be accurate and up to date.
```

Note that I have another suggestion to add the dashboard-privacy link to the footer further down in this review.

The spaces between the link text and footer ID are breaking the links.

File: site/en/docs/webstore/best_practices/index.md [hunk 5]
```diff
@@ -9,10 +9,33 @@ description: >
...
 matures and we learn from developers' experiences, these recommendations will be updated.
 
-## Design a great app
+## Design a high-quality extension
+
+We encourage you to develop extensions that are of high quality. High-quality extensions conform to
+standards of performance, security, and user experience, as summarized by the following guidelines:
+
+* **Compliance:** Does the extension comply with our [program policies]? Extensions must not violate
+any of these policies.
```
Comment (dotproto) [inline site/en/docs/webstore/best_practices/index.md:18]: That's a fancy bit of shorthand. I didn't know we could write links this way 😳

File: site/en/docs/webstore/best_practices/index.md [hunk 6]
```diff
@@ -54,162 +76,31 @@ See [Supplying Images][8] for guidelines on the images you should supply to the
...
 [5]: /docs/webstore/check_for_payment#cache
 [6]: #top
 [7]: /docs/webstore/branding
 [8]: /docs/webstore/images
 [9]: #top
+[mv3-overview]: /docs/extensions/mv3/intro/mv3-overview/
+[program policies]: /docs/webstore/program_policies/
+[user-daat-faq]: /docs/webstore/user_data/
+[supplying-images]: /docs/webstore/images/
+[completing-listing]: /docs/webstore/cws-dashboard-listing/
```
Comment (dotproto) [inline site/en/docs/webstore/best_practices/index.md:106]:
```suggestion
[completing-listing]: /docs/webstore/cws-dashboard-listing/
[dashboard-privacy]: /docs/webstore/cws-dashboard-privacy/
[mv3-overview]: /docs/extensions/mv3/intro/mv3-overview/
[program policies]: /docs/webstore/program_policies/
[supplying-images]: /docs/webstore/images/
[user-data-faq]: /docs/webstore/user_data/
```

Fixes a typo in "user-data-faq", adds a "dashboard-privacy" link,  and sorts the links alphabetically.

**Review by dotproto — CHANGES_REQUESTED** [review]
This draft does not touch the "Support Google Accounts" section, but it at least needs to be updated to remove references to Chrome Apps and purchasing items from the Chrome Web Store.

I think we should also update that section to pivot it's stance a little to say something along the lines of, 'if your extension supports user accounts, we recommend (but do not require) that you also support logging in from Google accounts. Many Chrome will already be logged into their accounts; reducing the number of logins improves the user's experience.'

File: site/en/docs/webstore/best_practices/index.md (hunk 5 above)
Comment (awfuchs) [reply site/en/docs/webstore/best_practices/index.md:18]: It's concise, but it leaves you with constraints like no uppercase...

File: site/en/docs/webstore/best_practices/index.md [hunk 7]
```diff
@@ -9,10 +9,33 @@ description: >
...
+* **Compliance:** Does the extension comply with our [program policies]? Extensions must not violate
+any of these policies.
+
+* **Manifest V3:** Is the extension built on Manifest V3? Manifest V3 reflects the aims of our
+  platform vision, and all High Quality extensions should use it. (See this
+  [MV3 overview][mv3-overview].)
+
+* **Security:** Is the extension safe for users? Make sure your extension does not pose security
+  threats, does not use deceptive installation tactics and meets Chrome Web Store's data privacy
+  requirements (see this [FAQ][user-data-faq]).
```
[the remark this answers is not in the document]
Comment (awfuchs) [reply site/en/docs/webstore/best_practices/index.md:26]: Done. There's now a separate bullet for user data privacy, and each of these (security bullet, user data bullet) links to the appropriate article.

File: site/en/docs/webstore/best_practices/index.md [hunk 8]
```diff
@@ -9,10 +9,33 @@ description: >
 This page has advice on how you should implement your app and list it in the store. As the store
 matures and we learn from developers' experiences, these recommendations will be updated.
 
-## Design a great app
+## Design a high-quality extension
+
+We encourage you to develop extensions that are of high quality. High-quality extensions conform to
+standards of performance, security, and user experience, as summarized by the following guidelines:
+
```
[the remark this answers is not in the document]
Comment (awfuchs) [reply site/en/docs/webstore/best_practices/index.md:16]: Done, see comment at line 26.

**Review by dotproto — APPROVED** [review]
Please update the updated date value in the frontmatter.

**Pull request merged**